lib/identity-provider.js

```diff
@@ -524,15 +531,77 @@ IdentityProvider.prototype.getGraph = function (uri, callback) {
     })
 }
 
-// Handle POST requests on account creation
-IdentityProvider.prototype.post = function (req, res, next) {
-  if (!req.body) {
+/**
+ * Sends a request to the OIDC Provider's Users API endpoint, to create a new
+ * user record with the provider.
+ * @method createOIDCUser
+ * @param oidcRpClient {OidcRpClient} Local/trusted OIDC client
+ * @param webId {String} WebID URL of the new user to be created
+ * @param options {Object} User options hashmap
+ * @param options.password {String} User's signin password. NOTE: Must be 8+
+ *   characters, mix of alpha and numeric
+ * @param [options.email] {String} User's email address (for recovery etc)
+ * @param [options.name] {String} User's name
+ * @return {Promise}
+ */
+IdentityProvider.prototype.createOIDCUser =
```

Comment:
createOIDCUser shouldn't be in the Identity provider package (maybe put that in the oidc one?)